### ISSUE TITLE
Couple thoughts...

### ISSUE THREAD

**Issue by DarwinJS** [chocolatey/chocolatey-test-environment#4]
Thanks so much for setting this up!

On this item: "Hyper-V may need to be disabled for Virtualbox to work properly if your computer is a Windows box." - I know you can't run VMWare and Hyper-V, but I've never had troubles with Virtual Box with Hyper-V enabled on Win 8.1 and 10.  Maybe if there have been known observed cases of conflict change this to "Although rare, in some cases Hyper-V may need to be disabled for Virtualbox to work properly on Windows."

Would be great to show the actual git clone command somewhere and give it a sample folder for the rest of the doc.  Like this:

git clone https://github.com/chocolatey/chocolatey-test-environment.git c:\users\public\chocolatey-test-environment

OR

git clone https://github.com/chocolatey/chocolatey-test-environment.git <chocotestenvfolder>

**Comment by ferventcoder** [comment]
:+1:

**Comment by dtgm** [comment]
>On this item: "Hyper-V may need to be disabled for Virtualbox to work properly if your computer is a Windows box."

Incorrect. Hyper-V **must** be disabled for Virtualbox to run this VM.

>I know you can't run VMWare and Hyper-V, but I've never had troubles with Virtual Box with Hyper-V enabled on Win 8.1 and 10. 

That is because your VMs were only 32-bit. Turn on Hyper-V, restart, and try to run a 64-bit VM. Won't/can't happen .... unless you sleep with an architectural hardware engineer and bathe in ES cpus...

>Maybe if there have been known observed cases of conflict change this to "Although rare, in some cases Hyper-V may need to be disabled for Virtualbox to work properly on Windows."

No rare test cases. Very simply because the test VM is 64-bit and Virtualbox must have access to the x86 CPU virtualization extensions (VT-x/AMD-V) to run a 64-bit VM.

**Comment by ferventcoder** [comment]
> bathe in ES cpus

Lulz

**Comment by ferventcoder** [comment]
@DarwinJS @dtgm other than HyperV, do we still need the git clone command?

**Comment by regexaurus** [comment]
This issue is pretty old, but hey it's still open, so thought I would add my .02. I needed (wanted) both Hyper-V and VirtualBox on my workstation, even before I started trying out chocolatey-test-environment, and ran into the conflict between the two some time ago. I ended using the workaround Scott Hanselman [nicely wrote up here](https://www.hanselman.com/blog/SwitchEasilyBetweenVirtualBoxAndHyperVWithABCDEditBootEntryInWindows81.aspx). It adds a boot (BCD) entry without hypervisor support. Now when I (re)boot my workstation, I can boot to an environment that supports Hyper-V and Docker OR to an environment that supports VirtualBox. I like rebooting about as much as anybody else, but its not so bad with SSDs.
For what it's worth.

**Comment by ziggythehamster** [comment]
You can avoid doing the BCD edit yourself with [this handy tool](https://unclassified.software/en/apps/hypervswitch). The README should definitely be updated; if Hyper-V is enabled (this is a boot-time option), VirtualBox basically doesn't work (it's x86 only and exceptionally slow because everything is emulated in software).

**Comment by ferventcoder** [comment]
I thought we put up a HyperV image you could use - I could be mistaken and we have not added that yet here (but it is available).
